test/extended/image_ecosystem/scl.go

```diff
@@ -44,10 +46,19 @@ func defineTest(image string, t tc, oc *exutil.CLI) {
 			}
 
 			g.By("checking the log of the pod")
-			log, err := oc.KubeClient().CoreV1().Pods(oc.Namespace()).GetLogs(pod.Name, &kapiv1.PodLogOptions{}).DoRaw()
+			err = wait.Poll(1*time.Second, 10*time.Second, func() (bool, error) {
```

Comment:
So going from one API hit to one a second for 10 seconds we go from N% failure to M% failure? We generally have crazy 10minute timeouts in e2es ... do we gain by making this timeout here 2minutes, 5minutes? Why 10s?

Reply:
I was reasonably confident 10s was enough.  In the flakes that I saw, the check failed, but then the pod state dumper (which dumps the logs from the pod) saw the value we were expecting.

I think there is a very small window here between the pod being running (which we already wait for) and the logs containing the string we expect (which should be pretty much immediate because it's just the usage output from the image, once the container starts running the output will be immediate).

So that's why.  You're right that it could be higher and if we keep seeing flakes I guess i'll raise, it, but i'm pretty sure 10s is all we need here.